WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1457,7 +1488,18 @@ private ReaderPostAdapter getPostAdapter() {
         if (mPostAdapter == null) {
             AppLog.d(T.READER, "reader post list > creating post adapter");
             Context context = WPActivityUtils.getThemedContext(getActivity());
-            mPostAdapter = new ReaderPostAdapter(context, getPostListType(), mImageManager);
+            mPostAdapter = new ReaderPostAdapter(context, getPostListType(), mImageManager, new NewsCardListener() {
```

Comment:
Just curious, any particular reason we are setting listener here in a constructor instead of something like `setNewsCardListener`, similar to other listeners?? :)

Reply:
I wanted to make the listener mandatory and final - but you are right, that the other listeners have same restrictions and they are being set using a setter -> I refactored it so it's consistent with other listeners:). Thanks for pointing this out!